UI/data/locale/en-US.ini

```diff
@@ -570,11 +570,16 @@ Basic.Settings.General.SystemTrayHideMinimize="Always minimize to system tray in
 Basic.Settings.General.SaveProjectors="Save projectors on exit"
 Basic.Settings.General.SwitchOnDoubleClick="Transition to scene when double-clicked"
 Basic.Settings.General.StudioPortraitLayout="Enable portrait/vertical layout"
+Basic.Settings.General.Multiview="Multiview"
+Basic.Settings.General.Multiview.MouseSwitch="Change scenes with clicks"
```

Comment:
I'd probably name this "Click to switch between scenes"

Reply:
Will update.